.gitignore

```diff
@@ -17,6 +17,10 @@ __pycache__/
 # C extensions
 *.so
 
+/inference/weights/*_weight
```

Comment:
good idea to put these here! Do you want to just remove the .gitignore file I added to the inference folder, and copy and paste the ignore directives from that file to here (and adjust the path)?